assets/multiqc_config.yml

```diff
@@ -1,7 +1,7 @@
 report_comment: >
-  This report has been generated by the <a href="https://github.com/nf-core/isoseq/1.1.5" target="_blank">nf-core/isoseq</a>
-  analysis pipeline. For information about how to interpret these results, please see the
-  <a href="https://nf-co.re/isoseq/1.1.5/output" target="_blank">documentation</a>.
+  This report has been generated by the <a href="https://github.com/nf-core/isoseq/tree/dev" target="_blank">nf-core/isoseq</a> analysis pipeline. For information about how to interpret
```

Comment:
bump version to version 2?

Reply:
Updated.

Reply:
nf-core lint is expecting having dev and not 2.0.0
Should revert to dev?

Reply:
And if you link to the release tag? Like:

https://github.com/nf-core/sarek/blob/5cc30494a6b8e7e53be64d308b582190ca7d2585/assets/multiqc_config.yml#L5-L8

Reply:
`nf-core lint` is still not happy ^^
https://github.com/nf-core/isoseq/actions/runs/10723006473/job/29735313846?pr=32#step:6:252

Reply:
One last try with `releases` in there, `https://github.com/nf-core/isoseq/releases/tag/2.0.0`? Looks like that should be it https://github.com/nf-core/tools/blob/930ece572bf23b68c7a7c5259e918a878ba6499e/nf_core/lint/multiqc_config.py#L102-L105

Reply:
I gave a try with 2.0.0dev and 2.0.0, still the same error :/

Reply:
Alright, not sure why. I'll approve and you can decide if you want to try further or revert back to using dev. I think it's nice to have the link to the actual version in the MultiQC report though.

Reply:
Thanks for your time and approval. 
I'm gonna leave it to 2.0.0, as it's the target version.